app/views/anonymous_feedback/_feedback.html.erb

```diff
@@ -15,6 +15,13 @@
     <br/>
     <strong>comment</strong>: <%= anonymous_contact.details %>
   <% end %>
+<% elsif anonymous_contact.type == "aggregated-service-feedback" %>
+  <strong><%= anonymous_contact.service_satisfaction_rating %> rating</strong>:
```

Comment:
done, Liz is out until Monday I think but could maybe check with a content person?